### PR TITLE
Add RPM debug option

### PR DESCRIPTION
**Pull request by haslinghuis**
See: https://github.com/betaflight/betaflight/pull/12562

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
i'll approve upon merge of https://github.com/betaflight/betaflight/pull/12562

**Review by McGiverGim — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
Need to create a new PR as it cannot be reopened.

**Pull request closed without merging**
